test/ModuleInterface/arm64e-fallback.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: not %target-typecheck-verify-swift -target arm64e-apple-ios13.0 -F %S/Inputs | %FileCheck %s
+
+// REQUIRES: OS=ios
+
+import DummyFramework
+// CHECK: could not find module 'DummyFramework' for target 'arm64e-apple-ios'; found: arm64-apple-ios, arm64
```

Comment:
Can you use `// expected-error` here, or does it not work?

Reply:
There's a fatal error, so `%target-typecheck-verify-swift` fails (so you need a `not`) and `expected-error` doesn't do what you'd expect (e.g. even if the `expected-error` doesn't match, the `not` makes the test pass).